app/lib/actions/pulp3/content_view_version/export.rb

```diff
@@ -0,0 +1,33 @@
+module Actions
+  module Pulp3
+    module ContentViewVersion
+      class Export < Pulp3::AbstractAsyncTask
+        input_format do
+          param :smart_proxy_id, Integer
+          param :content_view_version_id, Integer
+          param :export_base_dir, String
+        end
+
+        def export(cvv)
+          smart_proxy = SmartProxy.find(input[:smart_proxy_id])
+          api = ::Katello::Pulp3::Api::Core.new(smart_proxy)
+          export_base_dir = input[:export_base_dir]
+          repository_hrefs = cvv.repositories.map { |repo| repo.repository_reference[:repository_href] }
+          exporter_data = api.exporter_api.create(name: "#{cvv.generate_exporter_id}-#{rand(9999)}",
+                                                  path: "#{export_base_dir}/#{cvv.generate_exporter_path}",
+                                                  repositories: repository_hrefs)
+          tasks = [api.export_api.create(exporter_data.pulp_href, {})]
+          output[:exporter_data] = exporter_data
+          output[:export_data] = api.export_api.list(exporter_data.pulp_href).results.first
```

Comment:
i don't know that you are guaranteed to be able to access export_data here.  Consider if all workers are busy syncing some repos or something else.   This should appear in the task as a 'created_resource', if it doesnt that is a pulp bug IMO.  Assuming it does, you can just pass the pulp_tasks to the tasks that need them and grab it out of created_resources there

Reply:
Hmm good point. I have not been able to reproduce this issue, but I can see it being possible. Let me see if I can make that issue go away :)

Reply:
I have updated the actions a bit. Resolve if you are happy with the outcome